src/torznab.ts

```diff
@@ -681,3 +662,76 @@ async function makeRequests(
 		candidates: results,
 	}));
 }
+async function getAndLogIndexers(
+	enabledIndexers: Indexer[],
+	timestampDataSql,
+	mediaType: MediaType,
+	searchee: Searchee,
+): Promise<{ indexersToUse: Indexer[]; parsedMedia?: ParsedMedia }> {
+	const { excludeRecentSearch, excludeOlder } = getRuntimeConfig();
+	const mediaTypeStr = mediaType.toUpperCase();
+
+	const indexersToUseByTimestamp = enabledIndexers.filter((indexer) => {
+		const entry = timestampDataSql.find(
+			(entry) => entry.indexerId === indexer.id,
+		);
+		if (!entry) {
+			return true;
+		}
+		return (
+			(!excludeOlder || entry.firstSearched > nMsAgo(excludeOlder)) &&
+			(!excludeRecentSearch ||
+				entry.lastSearched < nMsAgo(excludeRecentSearch))
+		);
+	});
+
+	const indexersToUse = indexersToUseByTimestamp.filter((indexer) => {
+		return indexerDoesSupportMediaType(
+			mediaType,
+			JSON.parse(indexer.categories),
+		);
+	});
+	const skippingSearching = (reason: string) =>
+		`Skipped searching on indexers for ${searchee.name} ${reason} [${mediaTypeStr}]`;
+
+	let skippingIndexersCallout =
+		enabledIndexers.length > indexersToUse.length ? " (filtered by " : "";
+
+	if (enabledIndexers.length > indexersToUseByTimestamp.length) {
+		skippingIndexersCallout +=
+			indexersToUseByTimestamp.length === indexersToUse.length
+				? "timestamps)"
+				: "timestamps ";
+	}
+	if (indexersToUseByTimestamp.length > indexersToUse.length) {
+		skippingIndexersCallout +=
+			enabledIndexers.length === indexersToUseByTimestamp.length
+				? "supported category)"
+				: "and supported category)";
+	}
+	if (indexersToUse.length === 0) {
+		logger.info({
+			label: Label.TORZNAB,
+			message: `${skippingSearching(skippingIndexersCallout)}`,
+		});
+		throw new Error("SKIPPED");
+	}
+
+	// Parse media from arrs
+	const res: Result<ParsedMedia, boolean> =
+		indexersToUse.length > 0
+			? await scanAllArrsForMedia(searchee, mediaType)
+			: resultOfErr(false);
+	const parsedMedia = res.isOk() ? res.unwrap() : undefined;
+	const ids = parsedMedia?.movie ?? parsedMedia?.series;
+	const idsStr = ids ? `: ${formatFoundIds(ids)}` : "";
+
+	const skippingIndexers = `Searching ${indexersToUse.length} indexers for ${searchee.name}${skippingIndexersCallout} [${mediaTypeStr}${idsStr}]`;
+	logger.info({
```

Comment:
this log line contains a lot of information. can some of it move to a followup verbose log?

Reply:
I like having these all in info so it's easier for users to catch something being wrong. I myself have caught bugs because something didn't parse as expected. If we are to split it up, I rather put `indexersToUse.length` and `skippingIndxersCallout` in verbose.

Reply:
sg